common/production_methods/07_government.txt

```diff
@@ -407,7 +407,128 @@ pm_religious_academia = {
 		}
 	}			
 }			
+pm_no_academic_focus = {
+	texture = "gfx/interface/icons/production_method_icons/unused/no_philosophy_dept.dds"
+}
+pm_military_academy = {
+	texture = "gfx/interface/icons/production_method_icons/unused/military_academy.dds"
+	building_modifiers = {
+		level_scaled = {
+			building_employment_officers_add = 150
+            building_employment_academics_add = 50
+            building_employment_soldiers_add = 250
+		}
+	}
+	building_modifiers = {
+		workforce_scaled = {
+			building_input_paper_add = 0.5
+			building_input_small_arms_add = 1
+		}	
+	}
+	country_modifiers = {
+		workforce_scaled = {
+			country_military_tech_spread_mult = 0.01
+		}
+	}
+}
+
+pm_research_institute = {
+	texture = "gfx/interface/icons/production_method_icons/unused/engineering_university.dds"
+	building_modifiers = {
+		level_scaled = {
+			building_employment_academics_add = 100
+            building_employment_engineers_add = 100
+            building_employment_machinists_add = 150
+            building_employment_laborers_add = 100
+		}
+	}
+	building_modifiers = {
+		workforce_scaled = {
+			building_input_paper_add = 1
+			building_input_services_add = 0.5
+		}	
+	}
+	country_modifiers = {
+		workforce_scaled = {
+			country_production_tech_spread_mult = 0.01
+		}
+	}
+}
+
+pm_liberal_arts = {
+	texture = "gfx/interface/icons/production_method_icons/unused/social_science.dds"
+	building_modifiers = {
+		level_scaled = {
+			building_employment_academics_add = 150
+            building_employment_bureaucrats_add = 50
+            building_employment_clerks_add = 100
+            building_employment_laborers_add = 100
+		}
+	}
+	building_modifiers = {
+		workforce_scaled = {
+			building_input_paper_add = 0.5
+			building_input_services_add = 1
+		}	
+	}
+	country_modifiers = {
+		workforce_scaled = {
+			country_society_tech_spread_mult = 0.01
+		}
+	}
+}
 
+pm_general_studies = {
+	texture = "gfx/interface/icons/production_method_icons/unused/lightbulbs.dds"
+	building_modifiers = {
+		
+		level_scaled = {
+			building_employment_academics_add = 100
+            building_employment_engineers_add = 50
+            building_employment_officers_add = 50
+            building_employment_soldiers_add = 100
+            building_employment_clerks_add = 100
+            building_employment_laborers_add = 100
+		}
+	}
+	building_modifiers = {
+		workforce_scaled = {
+			building_input_paper_add = 0.5
+			building_input_small_arms_add = 0.5
+			building_input_services_add = 0.5
+		}	
+	}
+	country_modifiers = {
+		workforce_scaled = {
+			country_tech_spread_mult = 0.0033
```

Comment:
I would like to see a 0.0033333. Just so you're not missing out on 1% of the innovation compared focus. (the extra 3s diminish the rounding error).

Reply:
Understood. Wasn't sure how that affected things but I'll add more.